doc/classes/VisualShaderNode.xml

```diff
@@ -11,6 +11,7 @@
 			<return type="Array">
 			</return>
 			<description>
+				Returns an [Array] contained possible default values for all of the input ports of a node in the form like {index, value}".
```

Comment:
Returns an [Array] _containing_ default values for all of the input ports of the node in the form {index, value, index, value}.